plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -33,7 +33,7 @@ func init() {
 		func(client clientset.Interface, config io.Reader) (admission.Interface, error) {
 			// NOTE: we do not provide informers to the registry because admission level decisions
 			// does not require us to open watches for all items tracked by quota.
-			registry := install.NewRegistry(client, nil)
+			registry := install.NewRegistry(nil, nil)
```

Comment:
@smarterclayton - we have experimented in openshift with having a registry for admission versus replenishment.  i think this is calling out the need to continue that split upstream more formally.  

* For the `Evaluator.Get()` function call, I think we only now use it in a unit test downstream, as its usage is no longer needed that the prevObject is passed into admission control.  I can remove `Evaluator.Get` when 1.6 opens up.
* For the `Evaluator.List()` function call, that is basically what becomes unique about a registry when running in admission versus the controller that requires the client.  I can think about a better factoring there, but just haivng a NewRegistryForAdmission() versus NewRegistryForReplenishment(client) would work for me.